### PR TITLE
Revised documentation on synapse specification slightly

### PR DESCRIPTION
**Pull request by heplesser**
I tried to clarify the guide on synapse specifications a bit and replaced "distributions" with "expressions", since our mechanism is far more general than random distributions.

### CODE REVIEW

File: doc/guides/connection_management.rst [hunk 1]
```diff
@@ -274,24 +276,26 @@ dictionary specifying the synapse parameters in more detail.
...
 
-Specifying the synapse properties in a single dictionary or as
-:ref:`collocated synapses <collocated_synapses>` allows to draw
-synaptic parameters from random distributions and combine such
-parameters flexibly. In addition to the key ``synapse_model``, the
-dictionary can contain specifications for ``weight``, ``delay``,
-``receptor_type`` (see :ref:`receptor-types` for details) and
-parameters specific to the chosen synapse model. The specification of
+If synapse properties are given as a dictionary, this it must include 
+the key ``synapse_model``, and may additionally specify ``weight``,
```
Comment (stinebuu) [inline doc/guides/connection_management.rst:284]: You do not need to specify `synapse_model`, if you do not, the properties of `static_synapse` is changed. There is also a `this` too many in this sentence ;)

**Review by stinebuu — CHANGES_REQUESTED** [review]
Thanks, @heplesser! I'm fine with this, but you have written that a synapse dictionary must contain `synapse_model`, and this is not necessary.

File: doc/guides/connection_management.rst (hunk 1 above)
Comment (heplesser) [reply doc/guides/connection_management.rst]: @stinebuu I just revised the text, hope it is correct now :).

**Review by stinebuu — APPROVED** [review]
Very nice!

**Review by sarakonradi — COMMENTED** [review]
Thanks! Just added a (very minor) suggestion.

File: doc/guides/connection_management.rst [hunk 2]
```diff
@@ -274,24 +276,26 @@ dictionary specifying the synapse parameters in more detail.
...
-synaptic parameters from random distributions and combine such
-parameters flexibly. In addition to the key ``synapse_model``, the
-dictionary can contain specifications for ``weight``, ``delay``,
-``receptor_type`` (see :ref:`receptor-types` for details) and
-parameters specific to the chosen synapse model. The specification of
+If synapse properties are given as a dictionary, it may include the keys
+``synapse_model`` (default `static_synapse`), ``weight`` (default 1.0), 
+``delay`` (default 1.0), and ``receptor_type`` (default 0, see :ref:`receptor-types` for details),
+as well as parameters specific to the chosen synapse model. The specification of
 all parameters is optional and unspecified parameters will take on the
```
Comment (sarakonradi) [inline doc/guides/connection_management.rst:287]:
```suggestion
all parameters is optional, and unspecified parameters will take on the
```

**Review by sarakonradi — APPROVED** [review]

**Pull request merged**